lib/project_copier.rb

```diff
@@ -35,6 +36,9 @@ def copy
       # to keep the translations system working with these copied resources
       setup_associated_primary_language_translations(copied_project)
 
+      # Creates Talk roles via background worker
+      TalkAdminCreateWorker.perform_async(copied_project.id)
```

Comment:
I can't remember. Is this a safe operation to run a sidekiq job within a `transaction`?

Reply:
Hm, probably not if the worker is reliant on the mutated state of the db, which this would be. I'll move the call outside the block. 👍